database/engine/README.md

```diff
@@ -6,52 +6,28 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/engine/
 
 # Database engine
 
-The Database Engine works like a traditional database. There is some amount of RAM dedicated to data caching and
-indexing and the rest of the data reside compressed on disk. The number of history entries is not fixed in this case,
-but depends on the configured disk space and the effective compression ratio of the data stored. This is the **only
-mode** that supports changing the data collection update frequency (`update_every`) **without losing** the previously
-stored metrics.
+The Database Engine works like a traditional database. It dedicates a certain amount of RAM to data caching and
+indexing, while and the rest of the data resides compressed on disk. Unlike other [database modes](/database/README.md),
```

Comment:
> while the rest

Reply:
`database modes` -> `memory modes`

Reply:
Fixed both of these (and the rest of yours, @zack-shoylev)!